bats/fb-content-katello.bats

```diff
@@ -82,7 +77,7 @@ setup() {
 @test "create activation key" {
   hammer -u admin -p changeme activation-key create --organization="Default Organization" \
     --name="Test AK" --content-view="Test CV" --lifecycle-environment="Test" \
-    --unlimited-hosts | grep -q "Activation key created"
+    --unlimited-content-hosts | grep -q "Activation key created"
```

Comment:
This was part of host unification, 6.2 should cherry-pick the right commits to get this `--unlimited-hosts` option which is probably a hammer-cli and a katello commit.  It doesn't exist anymore upstream, which is unfortunate breakage in hammer interface compatibility, but reverting it to `--unlimited-content-hosts`  will definitely break bats now